corehq/apps/users/tasks.py

```diff
@@ -41,6 +42,13 @@ def bulk_upload_async(domain, user_specs, group_specs, location_specs):
         'messages': results
     }
 
+
+def run_case_post_save_signals(docs):
+    for doc in docs:
+        case = CommCareCase.wrap(doc)
+        case_post_save.send(CommCareCase, case=case)
```

Comment:
and actually you can then make a reminder-specific function that just handles the deletion and pull that out of the processing function which also seems cleaner